test/python/sdpa/random_config.py

```
@@ -63,12 +63,20 @@ def compute_default_BHSD_strides(shape):
+def compute_packed_strides(shape, token_gap=0):
+    """Compute packed (ragged) BSHD strides for BHSD shape: (s*h*d, d, h*d, 1).
+
+    ``token_gap`` widens the token stride to ``h*d + token_gap`` elements —
+    the layout of a tensor VIEW into a larger per-token record. With
+    ``token_gap == h*d`` this is exactly a K or V view of a kv-interleaved
+    ``[T, 2, H, D]`` buffer (token stride ``2*h*d``), the layout
+    ``torch.nn.attention.varlen`` users produce by slicing a fused KV
+    projection."""
     if shape is None:
         return None
     b, h, s, d = shape
```

Ruff (0.16.1)

[warning] Unpacked variable `b` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)